python/dgl/data/__init__.py

```diff
@@ -70,6 +70,14 @@
     RomanEmpireDataset,
     TolokersDataset,
 )
+
+# RDKit is required for OGB-LSC PCQM4Mv2 and the datasets derived from it.
+# Exception handling was added to prevent crashes for users who are using other
+# datasets.
```

Comment:
I didn't quite get this comment. The added dataset is peptides-struct instead of PCQM4Mv2, and the RDKit seems not explicitly used.

Reply:
The peptides-struct is derived from PCQM4Mv2, I will fix it